### ISSUE TITLE
it's possible to return raw array insted of numbered elements?

### ISSUE THREAD

**Issue by tdias25** [rakit/validation#36]
i have a field called **email** that is an array with multiple emails,when i print the getValidData the result is:

```
Array
(
    [email.0] => a@a.com
    [email.1] => b@b.com
)
```

it's possible to only return the 'raw' array of the valid emails?

something like this:

```
[email] => Array
        (
            [0] => a@a.com,
            [1] => b@b.com
        )

```

used this rule:

`'email.*' => 'required|email'`

**Comment by emsifa** [comment]
No it's not possible right now. But we can provide that using [`Rakit\Validation\Helper::arraySet`](https://github.com/rakit/validation/blob/master/src/Helper.php#L120).

But I should have done my deadline right now. I would add this after that.

**Comment by emsifa** [comment]
This fixed in `v0.17.1` (#38). I will close this. You can open it again if it doesn't works.
